### PR TITLE
0.4.0 patch

### PR DESCRIPTION
**Pull request by Jiayuan-Gu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,9 +130,9 @@ env.close()
 
 Each `mani_skill2` environment supports different **observation modes** and **control modes**, which determine the **observation space** and **action space**. They can be specified by `gym.make(env_id, obs_mode=..., control_mode=...)`.
 
-The supported observation modes are `pointcloud`, `rgbd`, `state_dict` and `state`. Note that for the Maniskill 2022 Challenge, only `pointcloud` and `rgbd` are permitted.
+The bascially supported observation modes are `pointcloud`, `rgbd`, `state_dict` and `state`. Extra observation modes are `pointcloud_robot_seg` and `rgbd_robot_seg`, which include robot segmentation masks in addition. Note that for the Maniskill 2022 Challenge, only `pointcloud` and `rgbd` (and their `robot_seg` versions) are permitted.
```
Comment (StoneT2000) [inline README.md]: Typo also we should write

The basic observation modes supported are ... Additional observation modes which include robot segmentation masks are `pointcloud_robot_seg` and `rgbd_robot_seg`.

Also change Maniskill 2022 challenge to either 2023 or cal it Maniskill2 challenge ( I recommend we call it maniskill2 challenge and make it more of a long running thing)

File: setup.py [hunk 2]
```diff
@@ -12,7 +12,7 @@ def read_requirements():
 
 setup(
     name="mani_skill2",
-    version="0.3.2",
+    version="0.3.3",
```
Comment (StoneT2000) [inline setup.py:15]: should be 0.4.0?

File: README.md [hunk 3]
```diff
@@ -2,13 +2,20 @@
...
 
-**Notes**: We are actively introducing new functionalities and improvements (e.g. new tasks and highly efficient system for visual RL). See the [roadmap](https://github.com/haosulab/ManiSkill2/discussions/30) for more details.
+ManiSkill2 is a unified benchmark for learning generalizable robotic manipulation skills powered by [SAPIEN](https://sapien.ucsd.edu/). **It features 20 out-of-box task families with 2000+ diverse object models and 4M+ demonstration frames**. Moreover, it empowers fast visual input learning algorithms
+so that **a CNN-based policy can collect samples at about 2000 FPS with 1 GPU and 16 processes on a workstation**. The benchmark can be used to study a wide range of algorithms: 2D & 3D vision-based reinforcement learning, imitation learning, sense-plan-act, etc.
+
+Please refer our [documentation](https://haosulab.github.io/ManiSkill2) to learn more information.
+
+We invite you to participate in the associated [ManiSkill 2022 challenge](https://sapien.ucsd.edu/challenges/maniskill/2022/) where the top teams will be awarded prizes.
```
Comment (StoneT2000) [inline README.md]: 2022?

File: README.md (hunk 1 above)
Comment (Jiayuan-Gu) [reply README.md]: I will correct the typo. I will call it ManiSkill2 challenge.

File: setup.py (hunk 2 above)
Comment (Jiayuan-Gu) [reply setup.py:15]: I will bump the version when I actually release 0.4.0 to pypi. That commit is where I commit for 0.3.3

File: README.md (hunk 3 above)
Comment (Jiayuan-Gu) [reply README.md]: Will change.

**Review by xuanlinli17 — APPROVED** [review]
